src/components/GlobalStyles.js

```diff
@@ -3,35 +3,36 @@ import { Global, css } from "@emotion/core"
 
 export default function GlobalStyles({ children }) {
   return (
-    <div>
-      <Global
-        styles={css`
-          @import url("https://fonts.googleapis.com/css2?family=Finger+Paint&display=swap");
-          @import url("https://fonts.googleapis.com/css2?family=Solway:wght@300;400;500;700;800&display=swap");
-          *,
-          *:before,
-          *:after {
-            box-sizing: border-box;
-            padding: 0;
-            margin: 5px;
-          }
-          html {
-            background: #111;
-            color: #fff;
-            font-family: "Solway", serif;
-            
-            
-          }
-          h1,
-          h2,
-          h3 {
-            font-family: "Finger Paint", cursive;
-            
-          }
-          
-        `}
-      />
-      {children}
-    </div>
+
+    <Global
+      styles={css`
+        @import url("https://fonts.googleapis.com/css2?family=Finger+Paint&display=swap");
+        @import url("https://fonts.googleapis.com/css2?family=Solway:wght@300;400;500;700;800&display=swap");
```

Comment:
You can import both fonts in a single request, which has positive effects to the performance:
```
@import url('https://fonts.googleapis.com/css2?family=Finger+Paint&family=Solway&display=swap');
```

And make sure if you need all different font weights. These font-weigths (300,400,500,700,800) have a big impact on the bandwidth.

Reply:
Oh okay, good to know. I will leave the font weights in for now for testing purposes, and add an issue to the project to check at the end if I really used them all.